cpp/dolfinx/la/SparsityPattern.h

```diff
@@ -67,6 +67,22 @@ class SparsityPattern
 
   /// @brief Insert non-zero locations using local (process-wise)
   /// indices.
+  void insert(std::int32_t row, std::int32_t col);
+
+  /// @brief Insert non-zero locations using local (process-wise)
+  /// indices.
+  ///
+  /// This routine inserts non-zero locations at the outer product of rows and
+  /// cols to the sparsity pattern, i.e. adds the matrix entries at
+  ///   A[row[i], col[j]] for all i, j
+  /// Enabling easy setting of dense sub-blocks, for example
```

Comment:
Not sure about the word dense. What if I pass e.g. `1, 3, 4` `1, 3, 4`.

Reply:
The introduced entries then form a dense 3x3 entries again, not locally dense in the sparse matrix, but a dense matrix worth of entries - not possible to only specify a sub set of these with this function.